agent-installation/kubernetes.mdx

```diff
@@ -116,7 +103,27 @@ The following section describes how to install and uninstall the Kubernetes Agen
   ```
 </Accordion>
 
-<Accordion title="Uninstall">
+# Troubleshooting
+<Warning>  Run only one MW Agent DaemonSet per cluster per host. Multiple agents will cause unexpected behavior. </Warning>
+
+Once the MW Agent is installed on your cluster metrics should begin appearing in the Middleware application within a few minutes. If metrics do not appear, confirm that Kubernetes has access to the internet, either directly or through a proxy, and that only one MW Agent is running per cluster. 
+
+For any issues please reach out to our team in your Slack channel or through the help menu in the Middleware Application.
+
+<Accordion title="Switching Between Contexts">
+If you are using another thirdy-party resource for mananging your Kubernetes Infrastructure (e.g. Lens), you must create cluster names that match the names of the contexts used by that third party resource. 
```

Comment:
I'd word this a little differently. The issue here is all user error really rather than anything related to Lens. What's happening is that they're naming their clusters both "kubernetes" and then managing prod vs. dev using Lens project names. 

I think the issue is the word "must". If it's a must it should be a step in the docs rather than troubleshooting. Better wording would be... (also typo in mananging that needs to be fixed)

"If you are using another thirdy-party resource for managing your Kubernetes Infrastructure (e.g. Lens) it's possible your cluster names are similar across clusters. This will make it hard to delineate between the clusters inside of Middleware.

To fix this, you can force a specific `cluster.name` by setting the `MW_KUBE_CLUSTER_NAME` environment variable..."

Note I also changed some of the wording on how to talk about cluster.name and the env variable.